addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -150,6 +150,17 @@ export class Link extends Component {
             this.$link.css('border-style', '');
             this.$link.css('border-color', '');
         }
+        // When multiple buttons follow each other, they may break on 2 lines
+        // or more on mobile, so they need a margin-bottom.
+        if (data.classes.includes("btn")) {
```

Comment:
Please, could you do something to avoid the repetition of this condition that already exists in the `applyLinkToDom` function.

Reply:
Am I blind? I don't think the condition already exists. There is a condition for the opposite (`if (!data.classes.split(' ').includes('btn'))`), but within that condition the `btn` class is potentially added, so I can't just do an else, I need a complete recheck.

Reply:
I only meant to put the `data.classes.includes("btn")` condition in a variable. In one case verify that it is `false` and in the other that it is `true`;)